mtgsqlive/json2sql.py

```diff
@@ -176,6 +176,8 @@ def build_sql_schema(sql_connection: sqlite3.Connection) -> None:
         "frameVersion TEXT,"
         "hand TEXT,"
         "hasFoil INTEGER NOT NULL DEFAULT 0,"  # boolean
+        "hasLeadershipSkills INTEGER NOT NULL DEFAULT 0,"  # boolean
```

Comment:
I'm dumb and this isn't actually a boolean...
```
                 "hasLeadershipSkills",
                {
                    "brawl": is_brawl,
                    "commander": is_commander,
                    "oathbreaker": is_oathbreaker,
                },
```

Reply:
is now `leadershipSkills`

Reply:
hmm, this makes it a little trickier since it's an array of data which is often put into its own table. But this doesn't really seem like something that necessitates its own table. My first thought is a comma delimited string of formats that are `True`. Any thoughts?